Source/WebCore/css/calc/CSSCalcTree+Simplification.cpp

```diff
@@ -1279,6 +1279,20 @@ std::optional<Child> simplify(Sign& root, const SimplificationOptions& options)
     );
 }
 
+std::optional<Child> simplify(Anchor& anchor, const SimplificationOptions& options)
+{
+    if (!options.conversionData || !options.conversionData->style())
+        return { };
+
+    // FIMXE: Evaluate the anchor.
+    bool isValid = !anchor.elementName.isNull() || !options.conversionData->style()->positionAnchor().isNull();
+    if (!isValid && anchor.fallback) {
+        // Replace the anchor node with the fallback node.
+        return WTFMove(anchor.fallback);
```

Comment:
Should this be `std::exchange` instead of `WTFMove`? If anyone ever reads from `anchor.fallback` after this point, then the answer is probably yes, but if all we do with it is destroy the anchor, then move is fine.